src/java/org/apache/cassandra/service/reads/DataResolver.java

```diff
@@ -226,6 +231,7 @@ private PartitionIterator resolveWithReadRepair(ResolveContext context,
 
     private PartitionIterator resolveWithReplicaFilteringProtection(E replicas, RepairedDataTracker repairedDataTracker)
     {
+        // TODO: Update comments once this has made it through review...
```

Comment:
Will provide more commentary and adjustments to the existing comments once we're through review of the actual changes...